streaming/src/main/scala/org/apache/gearpump/streaming/task/Task.scala

```diff
@@ -171,6 +171,8 @@ abstract class Task(taskContext : TaskContext, userConf : UserConfig) extends Ta
 
   implicit val self = taskContext.self
 
+  val taskConfig = userConf
```

Comment:
Another approach is to define HBase as:

trait HBase {
  this: Task =>
...
  
 def userConf

  val pipelineConfig = userConf.getValue[PipelineConfig](PIPELINE)